javaslang-benchmark/src/test/java/javaslang/collection/VectorBenchmark.java

```diff
@@ -43,16 +47,17 @@
 
     public static void main(String... args) {
         JmhRunner.runDebugWithAsserts(CLASSES);
-        JmhRunner.runNormalNoAsserts(CLASSES, JAVA, FUNCTIONAL_JAVA, PCOLLECTIONS, ECOLLECTIONS, CLOJURE, SCALA, JAVASLANG);
+        JmhRunner.runNormalNoAsserts(CLASSES, /*FUNCTIONAL_JAVA, PCOLLECTIONS, ECOLLECTIONS,*/ JAVA, CLOJURE, SCALA, JAVASLANG);
     }
 
     @State(Scope.Benchmark)
     public static class Base {
-        @Param({ "32", "1024", "32768" /*, "1048576", "33554432", "1073741824" */ }) /* i.e. depth 1,2,3(,4,5,6) for a branching factor of 32 */
+        @Param({ "100", "10000" })
```

Comment:
Looks all good to me. +1 for trying parallel tests - should be a flag in the maven surefire plugin.